.eslintrc

```diff
@@ -1,4 +1,7 @@
 {
 	"env": { "browser": true },
-	"extends": [ "plugin:@wordpress/eslint-plugin/recommended-with-formatting" ]
+	"extends": [ "plugin:@wordpress/eslint-plugin/recommended-with-formatting" ],
+	"settings": {
+		"jest": { "version": "latest" }
+	}
```

Comment:
TODO: Perhaps pull this out to another PR.  My IDE was not properly running eslint without this change for some reason.

Reply:
I'm fine with this in this PR.